lib/pages/moviedetail/components/tab_item.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import '../../../gen/colors.gen.dart';
+
+class TabItem extends StatelessWidget {
```

Comment:
is this used?

Reply:
Yes. Because the cast and more tabs have the same UI